### PR TITLE
Fix controller status command

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: controller/controller_tool/tool/src/py/ControllerGUI.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def event_start_button_clicked(self):
         """Publish command."""
         command = DdsRecorderControllerCommand.EVENT
         args = Controller.command_arguments_to_string(
-            Controller.argument_change_state(DdsRecorderStatus.RECORDING))
+            Controller.argument_change_state(DdsRecorderStatus.STARTED))
```
Comment (jparisu) [inline controller/controller_tool/tool/src/py/ControllerGUI.py:306]: RUNNING

File: controller/controller_tool/tool/src/py/Controller.py [hunk 2]
```diff
@@ -128,67 +127,67 @@ def is_valid_dds_domain(self, dds_domain):
     def init_dds(self, dds_domain):
         """Initialize DDS entities."""
         factory = fastdds.DomainParticipantFactory.get_instance()
-        participant_qos = fastdds.DomainParticipantQos()
-        factory.get_default_participant_qos(participant_qos)
-        participant_qos.name('DDSRecorderController')
+        self.participant_qos = fastdds.DomainParticipantQos()
```
Comment (jparisu) [inline controller/controller_tool/tool/src/py/Controller.py:130]: why to store qos in a class variable?

File: controller/controller_tool/tool/src/py/Controller.py [hunk 3]
```diff
@@ -128,67 +127,67 @@ def is_valid_dds_domain(self, dds_domain):
...
+        factory.get_default_participant_qos(self.participant_qos)
+        self.participant_qos.name('DDSRecorderController')
```
Comment (jparisu) [inline controller/controller_tool/tool/src/py/Controller.py:132]: NIT:
It is not pythonic to declare class attributes outside __init__ . 
Some linters would argue about this practice:
https://vald-phoenix.github.io/pylint-errors/plerr/errors/classes/W0201.html

**Review by jparisu — CHANGES_REQUESTED** [review]

File: controller/controller_tool/tool/src/py/Controller.py (hunk 2 above)
Comment (juanlofer-eprosima) [reply controller/controller_tool/tool/src/py/Controller.py:130]: Some of these needed to be stored in object, I did not test exactly which ones. But indeed, in particular this one is not required.

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**